pilot/pkg/security/authz/builder/fuzz_test.go

```diff
@@ -48,7 +52,11 @@ func FuzzBuildTCP(f *testing.F) {
 		}
 		policies := push.AuthzPolicies.ListAuthorizationPolicies(selectionOpts)
 		option := fuzz.Struct[Option](fg)
-		New(bundle, push, policies, option).BuildTCP()
+		b := New(bundle, push, policies, option)
+		if b == nil {
+			fg.T().Skip()
+		}
+		b.buildTCP()
```

Comment:
```		
b.buildTCP()
```

Why `buildTCP() instead of BuildTCP()`? I notice they both exist, but it's not clear to me why we use `BuildHTTP()` in the former case, but not `BuildTCP()` in this case, when they seem to be analogous.

Reply:
typo :+1:. Thanks!